### PR TITLE
Add jasp-stats/jaspBSTS to jasp-desktop as a submodule

### PR DESCRIPTION
**Pull request by vandenman**
Related to https://github.com/jasp-stats/INTERNAL-jasp/issues/1617

Note that the module is not completely finished yet (e.g., there are no unit tests), but @petersen-f is working on that.

### CODE REVIEW

File: Desktop/mainwindow.cpp [hunk 1]
```diff
@@ -508,7 +508,7 @@ void MainWindow::loadQML()
 	//Load the ribbonmodel modules now because we have an actual qml context to do so in.
 	_ribbonModel->loadModules(	
 		{ 	"jaspDescriptives", "jaspTTests", "jaspAnova", "jaspMixedModels", "jaspRegression", "jaspFrequencies", "jaspFactor" },
-		{ 	"jaspAudit", "jaspBain", "jaspCircular", "jaspDistributions" , "jaspEquivalenceTTests", "jaspJags", "jaspLearnBayes", "jaspMachineLearning",
+		{ 	"jaspAudit", "jaspBain", "jaspCircular", "jaspDistributions" , "jaspEquivalenceTTests", "jaspJags", "jaspLearnBayes", "jaspMachineLearning", "jaspBsts",
```
Comment (JorisGoosen) [inline Desktop/mainwindow.cpp]: I thought we were doing the modules in alphabetical order?

**Comment by JorisGoosen** [comment]
So this all loos fine except for the order thing, this has to go in for 0.16.1 right?

**Review by JorisGoosen — CHANGES_REQUESTED** [review]
Sorry it is very nitpicky but if you can change the place where you inserted `jaspBsts` then this can be merged.

**Review by JorisGoosen — APPROVED** [review]

**Comment by JorisGoosen** [comment]
Still no unit-tests though right?

Well, I see that some new work has been done in https://github.com/petersen-f/jaspBSTS but as the unittests (and perhaps other points) are still missing from jasp-stats/jaspBsts I think it is better to wait until those are in.

Especially given the upcoming feature stop.

**Comment by vandenman** [comment]
@JorisGoosen happy to merge whenever you are! The unit tests are now fixed in the module.

**Review by JorisGoosen — APPROVED** [review]

**Comment by JorisGoosen** [comment]
Aaah, I wanted to merge this but I think it is better to wait until the meeting tomorrow where we discuss 0.16.2

**Comment by vandenman** [comment]
Didn't even know we were planning 0.16.2 but sounds like a good plan!

**Comment by EJWagenmakers** [comment]
Definitely a good plan!

**Comment by vandenman** [comment]
closed in favor of https://github.com/jasp-stats/jasp-desktop/pull/4764

**Pull request closed without merging**